pkg/webui/containers/fetch-table/index.js

```diff
@@ -270,7 +273,7 @@ class FetchTable extends Component {
     if (Boolean(getItemPathPrefix)) {
       entityPath = getItemPathPrefix(item)
     } else {
-      const item_id = item.id || item.ids[`${entitySingle}_id`]
+      const item_id = item.id || item[`${entitySingle}_id`] || item.ids[`${entitySingle}_id`]
```

Comment:
Let's avoid more arbitrary additions here and use the `getItemPathPrefix` instead for any non-standard id scenarios.